lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py

```diff
@@ -5,7 +5,7 @@
 
 import numpy as np
 
-from matplotlib import _api
+from matplotlib import _api, transforms as mtransforms
```

Comment:
Same aliasing question as above. Also, two lines below are two classes imported. Should we just add the function there or remove them and qualify the classes instead? I'm not saying we need general rules for this, but mixing the styles within a module is maybe not necessary?

Reply:
Switched to mpl.transforms, per the comment above.